client/signup/steps/jpo-summary/index.jsx

```diff
@@ -107,35 +107,35 @@ class JPOSummaryStep extends React.Component {
 								<ul className="jpo-summary__completed-onboarding">
 									{
 										jpoSiteTitle
-											? <li>{ checkMark } { translate( 'Site Title & Description' ) }</li>
+											? <li>{ checkMark } <span>{ translate( 'Site Title & Description' ) }</span></li>
 											: null
 									}
 									{
 										jpoSiteType
-											? <li>{ checkMark } { translate( 'Type of Site' ) }</li>
+											? <li>{ checkMark } <span>{ translate( 'Type of Site' ) }</span></li>
 											: null
 									}
 									{
 										jpoHomepage
-											? <li>{ checkMark } { translate( 'Type of Homepage' ) }</li>
+											? <li>{ checkMark } <span>{ translate( 'Type of Homepage' ) }</span></li>
 											: null
 									}
 									{
 										jpoContactForm
-											? <li>{ checkMark } { translate( 'Contact Us Form' ) }</li>
+											? <li>{ checkMark } <span>{ translate( 'Contact Us Form' ) }</span></li>
```

Comment:
Hi! I've found a possible matching string that has already been translated **31** times:
`translate( 'contact us' )` *ES Score: 7*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/jpo-final-step-cleanup)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).